src/Microsoft.Azure.WebJobs.Host/Tables/CloudTableArgumentBindingProvider.cs

```diff
@@ -1,48 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.IO;
-using System.Reflection;
-using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs.Host.Bindings;
-using Microsoft.Azure.WebJobs.Host.Storage.Table;
-using Microsoft.WindowsAzure.Storage.Table;
-
-namespace Microsoft.Azure.WebJobs.Host.Tables
-{
-    internal class CloudTableArgumentBindingProvider : IStorageTableArgumentBindingProvider
-    {
-        public IStorageTableArgumentBinding TryCreate(ParameterInfo parameter)
-        {
-            if (parameter.ParameterType != typeof(CloudTable))
-            {
-                return null;
-            }
-
-            return new CloudTableArgumentBinding();
-        }
-
-        private class CloudTableArgumentBinding : IStorageTableArgumentBinding
-        {
-            public Type ValueType
-            {
-                get { return typeof(CloudTable); }
-            }
-
-            public FileAccess Access
-            {
-                get
-                {
-                    return FileAccess.ReadWrite;
-                }
-            }
-
-            public async Task<IValueProvider> BindAsync(IStorageTable value, ValueBindingContext context)
-            {
-                await value.CreateIfNotExistsAsync(context.CancellationToken);
```

Comment:
It's moved to the converters.  There's some decent existing test coverage on this too.